packages/block-library/src/cover/transforms.js

```diff
@@ -51,6 +51,55 @@ const transforms = {
 					]
 				),
 		},
+		{
+			type: 'block',
+			blocks: [ 'core/group' ],
+			isMatch: ( { backgroundColor, gradient, style } ) => {
+				/*
+				 * Make sure the group has a background, since if the cover block is
```

Comment:
Maybe a rewording here would be good - something like:

>Make this transformation available only if the Group has background or gradient set, because otherwise `Cover` block displays a Placeholder. This helps us avoid arbitrary decisions about the Cover block's background and user confusion about the existence of previous content.